ssh/tunnel.go

```diff
@@ -177,6 +179,19 @@ func (t *Tunnel) Stop() error {
 	return nil
 }
 
+func (t *Tunnel) startKeepalive(user string, sshConnection *ssh.Client) {
```

Comment:
I'm not clear why we're passing in the user param here. It doesn't seem to be used other than in the log message.

Reply:
All the other log messages have the user, and I didn't want this one to be different just because it was extracted into a function.

Reply:
Fair point.

Reply:
Now that I look I can see that only two of the other logs actually report the user. I'm not really sure whether it's a useful thing to have in the log message, but consistency was what I was going for. Happy to remove if you think it's neater / doesn't provide value in the logs.